tardis/tardis_portal/api.py

```diff
@@ -872,3 +884,20 @@ def hydrate(self, bundle):
             bundle.data['file_object'].close()
             del(bundle.data['file_object'])
         return bundle
+
+
+class ObjectAclResource(MyTardisModelResource):
+    content_object = GenericForeignKeyField({
+        Experiment: ExperimentResource,
+        # ...
+    }, 'content_object')
+
+    class Meta:
+        authentication = default_authentication
+        authorization = ACLAuthorization()
+        queryset = ObjectACL.objects.all()
+        filtering = {
+            'object_id': ('exact', ),
```

Comment:
Carlo's method is fine, at least it fails when not using experiments, and removing it from filters is fine, too. However, you still need to set it when you create a new ObjectACL object, so you might want to adopt Carlo's method?